wasmCloud/wasmCloud
# feat: add wasi-stderr impl

## Feature or Problem

Add `wasi-stderr` impl to runtime host so we can move towards supporting WASI components.

## Related Issues

N/A

## Release Information

`next`

## Consumer Impact

Consumers can run WASI components which rely on `wasi-stderr`

## Testing

Built on platform(s)

Tested on platform(s)

### Unit Test(s)


### Acceptance or Integration


### Manual Verification

1. Prepare a WASI component (`wasm-tools component new ..`)
2. Sign the component with a local build of `wash` from main (which supports signing components)
3. Attempt to execute an Actor call 

```console
echo '' | cargo run -- /path/to/your/component_s.wasm Any.Any
```
Instead of seeing an error about `wasm-stderr` being the wrong type, you should see one about `wasi-io` being the wrong type
